pages/blog/posts/GSOD-enhancing-json-schema-docs.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Enhancing JSON Schema documentation: insights and progress from the Google Season of Docs 2024 project'
+date: '2024-09-03'
+tags:
+  - Google Season of Docs
+  - Documentation
+  - Community
+type: Documentation
+cover: img\posts\2024\GSOD-enhancing-json-schema-docs\JSON-website-cover.jpg
```

Comment:
This is not the url of the image you added. In addition, please use PNG for better performance.